tutorialposts/2021-10-14-vanilla-gan.md

```diff
@@ -2,7 +2,6 @@
 title = "Generative Adversarial Networks"
 published = "14 October 2021"
 author = "Ralph Kube"
-external = "http://fluxml.ai/Flux.jl/stable/tutorials/2021-10-14-vanilla-gan/"
```

Comment:
`external` makes the link in `/tutorials` work (attaches an external link to the bullets)